Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -19,6 +19,7 @@
 #include "InputCommon/ControllerInterface/OSX/OSX.h"
 #include "InputCommon/ControllerInterface/Quartz/Quartz.h"
 #endif
+#undef CIFACE_USE_SDL
```

Comment:
leftover from debugging?

Reply:
i haven't debugged the SDL problem yet. this PR is (hopefully) to help fix https://bugs.dolphin-emu.org/issues/13006

Reply:
But surely you aren't planning to disable SDL for every single platform?

Reply:
sometimes we can only dream